### PR TITLE
Fixes #5708 Add Appointment FHIR,Portal endpoints (#5709)

### LINKED ISSUE #5708: Add endpoints for Appointments to FHIR and patient portal apis
We currently only allow providers access to the appointment resource if they access the api as a provider.  We should add support to allow appointments to be retrieved via FHIR as well as with the standard api for the patient portal as a patient.  This will allow API consumers to provide display of appointments and facilitate appointment reminders for patient standalone applications.

### PR DESCRIPTION
**Pull request by adunsulag**
Backports fix in master for #5708 to rel-700.

### CODE REVIEW

**Comment by adunsulag** [comment]
If this looks good I'll bring this in as it's blocking my other PR.

File: version.php [hunk 1]
```diff
@@ -28,7 +28,9 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 470;
+
+$v_database = 472;
```
Comment (sjpadgett) [inline version.php:32]: Are you skipping because of my PR? 471

**Review by sjpadgett — APPROVED** [review]
looks okay to me. Ya been a busy bee my friend!

File: version.php (hunk 1 above)
Comment (sjpadgett) [reply version.php:32]: Don't worry about my backport as I can fix the conflicts from this.

File: sql/database.sql [hunk 2]
```diff
@@ -7310,9 +7310,11 @@ CREATE TABLE `openemr_postcalendar_events` (
   `pc_sendalertemail` VARCHAR( 3 ) NOT NULL DEFAULT 'NO',
   `pc_billing_location` SMALLINT (6) NOT NULL DEFAULT '0',
   `pc_room` varchar(20) NOT NULL DEFAULT '',
+  `uuid` binary(16) DEFAULT NULL,
   PRIMARY KEY  (`pc_eid`),
   KEY `basic_event` (`pc_catid`,`pc_aid`,`pc_eventDate`,`pc_endDate`,`pc_eventstatus`,`pc_sharing`,`pc_topic`),
-  KEY `pc_eventDate` (`pc_eventDate`)
+  KEY `pc_eventDate` (`pc_eventDate`),
+  UNIQUE KEY `uuid` (`uuid`)
```
Comment (bradymiller) [inline sql/database.sql:7317]: also needs corresponding sql upgrading elements to sql/patch.php

File: version.php (hunk 1 above)
Comment (bradymiller) [reply version.php:32]: @sjpadgett , @adunsulag 
note no need to increment this in the patching branches. it's ok to do it, but not used, so also ok not to do it :)

Comment (adunsulag) [reply version.php:32]: Had a conflict so I just bumped it to the higher version of the conflict.  Sounds like it doesn't really matter.

File: sql/database.sql (hunk 2 above)
Comment (adunsulag) [reply sql/database.sql:7317]: I added the database changes to patch.php.

**Review by bradymiller — APPROVED** [review]

**Pull request merged**